sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -615,6 +623,7 @@ public virtual async Task StopProcessingAsync(CancellationToken cancellationToke
         {
             cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
             await RunningTaskSemaphore.WaitAsync().ConfigureAwait(false);
+            Logger.EventProcessorStop(Identifier);
```

Comment:
Should we log this before the preceding wait? If, for some reason, there was an unusually long wait here, the event might be a good clue.